src/requirements.txt

```diff
@@ -1,20 +1,50 @@
 atomicwrites==1.2.1
 attrs==18.2.0
+Babel==2.6.0
 certifi==2018.8.24
 chardet==3.0.4
+Click==7.0
 coverage==4.5.1
-flake8==3.5.0
+datapunt-config-loader==1.1.0
 -e git+https://github.com/Amsterdam/objectstore.git@v1.0#egg=objectstore
+debtcollector==1.20.0
+flake8==3.5.0
+GeoAlchemy2==0.5.0
+geomet==0.2.0.post2
+-e git+https://github.com/Amsterdam/GOB-Core.git@v0.4.13#egg=gobcore
 idna==2.7
+iso8601==0.1.12
+jsonschema==2.6.0
+keystoneauth1==3.11.1
 mccabe==0.6.1
+monotonic==1.5
 more-itertools==4.3.0
+msgpack==0.5.6
+netaddr==0.7.19
+netifaces==0.10.7
+os-service-types==1.3.0
+oslo.config==6.6.2
+oslo.i18n==3.22.1
+oslo.serialization==2.28.1
+oslo.utils==3.37.1
 pbr==4.2.0
+pika==0.12.0
 pluggy==0.7.1
 py==1.6.0
 pycodestyle==2.3.1
 pyflakes==1.6.0
+pyparsing==2.3.0
 pytest==3.7.4
 pytest-cov==2.5.1
+python-keystoneclient==3.18.0
+python-swiftclient==3.6.0
+pytz==2018.7
+PyYAML==3.13
 requests==2.20.0
+rfc3986==1.1.0
+Shapely==1.6.4.post2
 six==1.11.0
+SQLAlchemy==1.2.13
```

Comment:
It looks like too many dependencies are declared.
Why is sqlaclchemy needed?

Reply:
SQLAlchemy is listed as a requirement on GOBCore. I've added all transitive requirements